problem2.py

```diff
@@ -0,0 +1,29 @@
+"""
+time: O(logn)
+space: O(1)
+Leet : Problem not from leetcode
+Problems faced: No problems.
+"""
+def infiniteBinary(nums,target):
+    slow = 0
+    fast = 1
+    
+    while nums[fast] < target:
```

Comment:
what if we reach out of range of the given array because here infinite also means a particular value right?

Reply:
You are right. My program doesn't handle that. I will review the problem and submit again soon. Good catch!